Tests/UITests/TooltipViewSpec.swift

```diff
@@ -37,84 +36,70 @@ class ToolTipViewSpec: QuickSpec {
         afterEach {
             mockServer.stop()
         }
+        func launchToolTip() {
```

Comment:
this name doesn't tell what kind of tooltip is launched. `displayAppLaunchTooltip` is better for example.
And also please add empty lines above and below this function for better readability.